libs/discordURL.ts

```diff
@@ -0,0 +1,30 @@
+export const DISCORD_OAUTH_URL = 'https://discord.com/api/oauth2/token';
+
+export const discordRedirectBaseUrl = `${process.env.NEXT_PUBLIC_DISCORD_REDIRECT_URI_BASE}/api/discord/auth`
+
+export const discordScope = ["identify", "guilds"].join(" ");
+
+export const DISCORD_CLIENT_ID = process.env.NEXT_PUBLIC_DISCORD_CLIENT_ID
+
+export const discordAuthUrl = (from: string) => {
+  const url = "https://discord.com/api/oauth2/authorize" +
+  "?client_id=1093511877813870592&" +
```

Comment:
Is this magic value or our credentials? Should this be replace by `DISCORD_CLIENT_ID`?

Reply:
updated to `DISCORD_CLIENT_ID` thank you